backend/src/main/java/dev/tripdraw/exception/auth/AuthExceptionType.java

```diff
@@ -1,13 +1,16 @@
 package dev.tripdraw.exception.auth;
 
 import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.FORBIDDEN;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
 
 import dev.tripdraw.exception.common.ExceptionType;
 import org.springframework.http.HttpStatus;
 
 public enum AuthExceptionType implements ExceptionType {
     NO_AUTH_HEADER(BAD_REQUEST, "Auth 헤더가 없습니다."),
-    ;
+    MEMBER_NOT_FOUND(NOT_FOUND, "존재하지 않는 회원입니다."),
+    AUTH_FAIL(FORBIDDEN, "접근 권한이 없습니다.");
```

Comment:
현재 코드에서 TripExeptionType의 경우 ;가 Enum아래 있는데요
해당 타입을 수정하면서 Auto Reformat 기능이 적용된 것 같아요.
따라서 AuthExceptionType, MemberExceptionType에 다음과 같이 변경해주면 좋을 것 같아요!
```suggestion
    MEMBER_NOT_FOUND(NOT_FOUND, "존재하지 않는 회원입니다."),
    AUTH_FAIL(FORBIDDEN, "접근 권한이 없습니다."),
    ;
```